tools/Makefile.sub

```diff
@@ -21,9 +26,14 @@ jlm-opt_SOURCES = \
 jlm-opt_LIBS = \
 	libtooling \
 	libllvm \
+	libmlir \
 	librvsdg \
 	libutil \
 
-jlm-opt_EXTRA_LDFLAGS = $(shell $(LLVMCONFIG) --libs core irReader) $(shell $(LLVMCONFIG) --ldflags) $(shell $(LLVMCONFIG) --system-libs)
+jlm-opt_EXTRA_LDFLAGS = \
+	$(shell $(LLVMCONFIG) --libs core irReader) \
+	$(shell $(LLVMCONFIG) --ldflags) \
+	$(shell $(LLVMCONFIG) --system-libs) \
```

Comment:
Could we shrink this into one command similarly to how it is in jlm/tooling/Makefile.sub

Reply:
All flags given to a single LLVMCONFIG